garage/tf/core/networks.py

```diff
@@ -0,0 +1,48 @@
+import tensorflow as tf
+
+
+def mlp(input_var,
```

Comment:
Please add docstrings for this function using the PEP287/PEP257 style discussed [here](https://github.com/rlworkgroup/garage/blob/master/CONTRIBUTING.md#garage-specific-python-style)